src/sass/formapp.scss

```diff
@@ -20,6 +20,13 @@ body {
     font-size: 18px;
   }
 }
+.bg-light {
+  background-color: #fff !important;
+}
+.card-body {
+  border: none;
+  padding: .375rem .75rem;
```

Comment:
why are these `rem`s instead of `6px 12px`?  Are there `rem`s elsewhere?

Reply:
probably because that was what was used elesewhere. I'll change.